app/controllers/ops_controller/settings/ldap.rb

```diff
@@ -328,6 +326,21 @@ def ls_delete
 
   private
 
+  def checked_or_params_id(klass)
+    objs = find_checked_items
+    obj = objs.blank? && params[:id] ? params[:id] : objs[0]
+    obj = from_cid(obj) if obj.present?
+    obj
+  end
+
+  def find_or_new(klass, ids)
```

Comment:
@martinpovolny Shouldn't the default value for id be `ids=nil`, in case you don't have any specific id?

Also: shouldn't it be `id`, since you are selecting just one item?